### ISSUE TITLE
Render States - Complexity

### ISSUE THREAD

**Issue by Ellisande** [garretttaco/learning-recompose#8]
I think Render States is a pretty good example, but the level of complexity jumps exponentially compared to the other two examples. I would either add an intermediate exercise or reduce the complexity of this one to be more linear.

**Comment by tylercollier** [comment]
I agree with Justin. For the workshop tomorrow, if you don't have time to change this by then, consider giving more hints about it as you lead the workshop.
